apis/broker/ts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@thecointech/broker-cad",
-  "version": "0.2.4",
```

Comment:
I don't know why this one went backwards, but as we don't publish this package (it's only used internally) it kinda doesn't matter.